Confus/Confus/MoveableWall.cpp

```diff
@@ -108,6 +108,7 @@ namespace Confus
         else
         {
             m_Transitioning = false;
+			m_MeshNode->setVisible(false);
```

Comment:
Didn't we discuss that this would break as soon as walls would transition up again?